pysnooper/tracer.py

```diff
@@ -198,10 +198,15 @@ class Tracer:
 
     You can also use `max_variable_length=None` to never truncate them.
 
+    Print time in elapsed time format::
+
+        @pysnooper.snoop(elapsed_time=True)
+
     '''
     def __init__(self, output=None, watch=(), watch_explode=(), depth=1,
                  prefix='', overwrite=False, thread_info=False, custom_repr=(),
-                 max_variable_length=100, normalize=False):
+                 max_variable_length=100, normalize=False,
```

Comment:
I think it's high time that this signature had one parameter per line, @cool-RR ?

Reply:
I don't like those too much. They take up too much vertical space.

Reply:
I'll fix the line wrapping later.